docs/source/coding_guidelines.rst

```diff
@@ -8,7 +8,7 @@ the repository.
 ClangFormat
 -----------
 
-ClangFormat (version 4.0) is used to check the C++ code formatting style in
+ClangFormat (version 5.0) is used to check the C++ code formatting style in
```

Comment:
should we even list the version here?

Reply:
I would think so.  Because the formatting end up (sigh) depending on the specific version.